tests/samples_test.py

```diff
@@ -0,0 +1,420 @@
+import pytest
+
+import numpy
```

Comment:
I think these should be ordered like this according to PEP 8
```suggestion
import numpy
import pytest
```